client/my-sites/plugins/plans/index.tsx

```diff
@@ -18,6 +18,7 @@ import PlansFeaturesMain from 'calypso/my-sites/plans-features-main';
 import { appendBreadcrumb } from 'calypso/state/breadcrumb/actions';
 import { getBreadcrumbs } from 'calypso/state/breadcrumb/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { MarketplaceFooter } from '../education-footer';
```

Comment:
```suggestion
import { MarketplaceFooter } from 'calypso/my-sites/plugins/education-footer';
```
I believe Calypso standardizes around absolute paths for imports. Probably need to do the same for MARKETPLACE_FLOW.

Once I did that locally, my IDE also suggested a change to the import order.

Reply:
Sounds good. Thanks for your quick review as well @DustyReagan!